services/ordering/src/EShopOnAbp.OrderingService.EntityFrameworkCore/Migrations/20220317123922_RemovedOrderStatusTable.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EShopOnAbp.OrderingService.Migrations
+{
+    public partial class RemovedOrderStatusTable : Migration
```

Comment:
When naming migrations or database interactions from code view; lets refer as `Entity` instead of _Table_.